### PR TITLE
[2] use lastest node version for dockerfile

### PR DESCRIPTION
**Pull request by FanchTheSystem**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,5 +16,5 @@ script:
   - npm install -g mocha
   - npm install -g nyc
   - cd $TRAVIS_BUILD_DIR/tests && docker-compose up -d
-  - cd $TRAVIS_BUILD_DIR && sleep 180 && docker ps && nyc --reporter=text mocha -- --timeout 10000
```
Comment (osallou) [inline .travis.yml]: why removing the timout on tests ?

Comment (osallou) [reply .travis.yml]: oh ok, I see it was moved to an option file

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM node:8-jessie
+FROM node:latest
```
Comment (osallou) [inline Dockerfile]: not fond of using latest.
May break at anytime when there is a  new release (not always backard compatible). Why not upgrading, but should specify a version.

Comment (FanchTheSystem) [reply Dockerfile]: f0659c1afcc957a1cecb343e0adf7a5d47fd50cb

**Comment by FanchTheSystem** [comment]
https://github.com/genouest/genouestaccountmanager/pull/46

**Comment by FanchTheSystem** [comment]
Fixed: https://github.com/genouest/genouestaccountmanager/commit/2381009a0743b6f4560e5b5daab133ddc41dd901

**Pull request closed without merging**